addons/mail/static/src/messaging/component/composer/composer.js

```diff
@@ -349,6 +337,7 @@ Object.assign(Composer, {
         hasCurrentPartnerAvatar: true,
         hasDiscardButton: false,
         hasFollowers: false,
+        hasMentionSuggestionsBelowPosition: false,
```

Comment:
You can remove this default as it will fall back on default of text input if you let it be undefined.

Reply:
Add optional: true

Reply:
Done

Reply:
Did you push it? I still see the default instead of optional

Reply:
Done (and pushed :p)